pyfr/solvers/base/elements.py

```diff
@@ -397,6 +397,10 @@ def get_norm_pnorms(self, eidx, fidx):
         fpts_idx = self.basis.facefpts[fidx]
         return self._norm_pnorm_fpts[fpts_idx, eidx]
 
+    def get_fpts(self, eidx, fidx):
```

Comment:
Is there a way to avoid adding this method?

Reply:
Well we need the access the flux points physical position. I could just write this function inside the `fluidforce` plugin if that simplifies things.

Reply:
I think that might be cleaner; get_fpts is also a bit ambiguous as it could also refer to the flux point array itself.

Reply:
Makes sense, I'll move this function to the plugin